getting-started-guide/glossary.mdx

```diff
@@ -0,0 +1,217 @@
+
+Traditionally, observability has been associated with three pillars:
+
+- Logs capture individual events or errors.
+- Metrics provide quantitative data over time, like CPU usage.
+- Traces represent workflows across microservices.
+
+However, modern observability expands on this by aggregating diverse data types from engineering, product, marketing, and security functions, all of which contribute to understanding the deeper “why” behind user interactions and system behaviors. This holistic view, in turn, enables real-time diagnostics, predictive analyses, and proactive issue resolution.
+
+In essence, observability transforms raw event data into actionable insights, helping organizations not only to answer “what happened?” but also to delve into “why it happened” and “what might happen next.”
+
+## P
+
+### Personal access token (PAT)
+
+See [Tokens](#token).
+
+### Playground
+
+The Axiom Playground is an interactive sandbox environment where you can quickly try out Axiom’s capabilities.
+
+To try out Axiom, go to the [Axiom Playground](https://play.axiom.co/).
+
+## Q
+
+### Query
+
+In Axiom, a query is a specific, structured request used to get deeper insights into your data. It typically involves looking for information based on defined parameters like keywords, date ranges, or specific fields. The intent of a query is precision: to locate, analyze, or manipulate specific subsets of data within vast data structures, enhancing insights into various operational aspects or user behaviors.
+
+Querying enables you to filter, manipulate, extend, and summarize your data. As opposed to [searching](#search) which relies on sampling, querying allows you to explore all your event data. For this reason, querying is the modern way of making sense of your event data.
+
+### Query-hours
+
+When you run queries, your usage of the Axiom platform is measured in query-hours. The unit of this measurement is GB-hours which reflects the duration (measured in milliseconds) serverless functions are running to execute your query multiplied by the amount of memory (GB) allocated to execution. This metric is important for monitoring and managing your usage against the monthly allowance included in your plan.
+
+For more information, see [Query costs](/reference/query-hours).
+
+## R
+
+### Role-based access control (RBAC)
+
+Role-based access control (RBAC) allows you to manage and restrict access to your data and resources efficiently.
+
+For more information, see [Access](/reference/settings#access-overview).
+
+## S
+
+### Search
+
+Most observability solutions rely on search to seek information within event data. In contrast, Axiom’s approach is [query](#query). Unlike search that only gives you approximate results because it relies on sampling, a query is precise because it explores all your data. For this reason, querying is the modern way of making sense of your event data.
```

Comment:
Interesting. I hadn’t seen a distinction like this before. What were your sources?